test/plugins/scss/index.test.js

```diff
@@ -10,6 +10,7 @@ const rimraf = require('rimraf');
 describe('scss resolver', () => {
   afterEach(() => {
     scss.clear();
+    rimraf.sync(path.join(outputDir + '/themes', '*'));
```

Comment:
The path call there should be `path.join(outputDir, 'themes', '*')`

Reply:
ok!